client/src/service/recipe/index.js

```diff
@@ -0,0 +1,21 @@
+import { getApiKey, getApiUrl } from '../index';
+import { handleError, pullOutJson } from '../handlers';
+
+export const getRecipeByIngredientsService = (ingredientsList, recipeCount) => {
+    const headers = new Headers({
+        'content-type': 'application/json',
+    });
+
+    const ingredients = ingredientsList.join(',+');
+    const params = `ingredients=${ingredients}&number=${recipeCount}&apiKey=${getApiKey()}`;
+    
+    return fetch(
+        encodeURI(`${getApiUrl()}/findByIngredients?${params}`),
+        {
+            method: 'GET',
+            headers,
+        },
+    )
+        .then(handleError)
```

Comment:
shouldn't there be a `.catch(err => //error handle code)` to handle the errors?

Reply:
Good catch! what do you think it should do when it catches an error?